src/log.js

```diff
@@ -540,6 +540,11 @@ export class Log {
    */
   prepareError_(error) {
     error = duplicateErrorIfNecessary(error);
+
+    // `associatedElement` is used to add the i-amphtml-error class; in
+    // `#development=1` mode, it also adds `i-amphtml-element-error` to the
+    // element and sets the `error-message` attribute.
+    error.associatedElement = error.messageArray?.find((item) => item?.tagName);
```

Comment:
`Array.p.find` is forbidden, because it's not available in IE11. 😭

Reply:
Removed!